### ISSUE TITLE
Leir III Sandworm

### ISSUE THREAD

**Issue by sqrw** [sqrw/sqrw.github.io#13]
Is this really confirmed not to happen? I could not find a source for this.

**Comment by StarCitizenTracker** [comment]
https://i.imgur.com/6EpWeDa.png

> "the sandworm was a real in game asset at one stage but nobody knows what happened to in the intervening years and its probably never coming back"

**Comment by Jehette** [comment]
@StarCitizenTracker What is that source even from? There is no way to verify that text based on that image.

**Comment by sqrw** [comment]
I agree with Jehette, here, the quote is no where to be found on google either so it's not in a public accessible space nor is it in spectrum/the forums.
